PrivateHeaders/APPX/ZIP.h

```diff
@@ -222,21 +262,28 @@ namespace appx {
         sink.Write(sizeof(data), data);
     }
 
+    // this writes [Content_Types].xml
     template <typename TSink>
     ZIPFileEntry WriteContentTypesZIPFileEntry(
-        TSink &sink, off_t offset,
+        TSink &sink, off_t offset, bool isBundle,
```

Comment:
You use a boolean here and in a few other places. Please avoid boolean parameters. Prefer an enum class or separate functions instead.